Cargo.toml

```diff
@@ -14,18 +14,25 @@ serde_json = "1.0.82"
 thiserror = "1.0.31"
 tokio = { version = "1.19.2", features = ["sync", "macros"] }
 tracing = "0.1.35"
-wasmtime = { version = "0.38.1", default-features = false, features = ["async"] }
+wasmtime = { version = "0.38.1", default-features = false, features = [
+  "async",
+] }
```

Comment:
I know I'm a bit picky, but that looks like unwanted changed from a code formatter. Mind rolling those back?

`git checkout -p upstream/main Cargo.toml` should help you roll back only part of this file

Reply:
Not picky at -- I'm learning :)
I'm using https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml 
It does a nice job of formatting toml, is that something you'd want to consider? or maybe it's not universal enough?

Reply:
reformatted to save some cycle time. LMK if you end up liking even-better-toml